.clj-kondo/rewrite-clj/rewrite-clj/config.edn

```diff
@@ -0,0 +1,5 @@
+{:lint-as
```

Comment:
What does this file do?

Reply:
This file was pulled in automatically by following the instructions in clj-kondo's project set up guide, which is here: https://github.com/clj-kondo/clj-kondo?tab=readme-ov-file#project-setup

To my understanding, this file (and the folder it is in) was written by the authors of this package clj-rewrite, which we have as a dependency, and they "exported" it so that projects using their package can import it and thus lint clj-rewrite correctly.

This seems to be the only one that exported a clj-kondo config. The super common ones like compojure are included in clj-kondo by default. The docs recommended adding this to version control. (see link above)

Reply:
Originally I wanted to incorporate this into the build, but when I saw only 1 dep came out of it, I figured it wasn't worth it.